docs/UserGuide.md

```diff
@@ -54,20 +54,37 @@ ContactsForGood (CFG) is a **desktop app for managing contacts, optimized for us
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
+* Each parameter is preceded by a **prefix**, which indicates the type of data the parameter represents.  
+  For example:
+    - `n/` is the prefix for `NAME`, as in `add n/John Doe`.
+    - `p/` is the prefix for `PHONE_NUMBER`, as in `add p/12345678`.
+
+* Any unrecognized parameter's prefix will be treated as part of the value for the preceding valid prefix. This means that care should be taken to use only the correct prefixes when entering commands to avoid unexpected behavior.
```

Comment:
Genuine question, do you think it will be good to also address the use of / here. For example, what will be the effect of doing something like:

t/NGO target/boss

Will boss be added as a tag? (Can we try this and make it clear here, thanks!)

Reply:
Nope tags must be alphanumeric and have no spacing too

Reply:
Hm might need to make this clearer too. Because commands like `add n/Benp/123e/2@gmail.coma/Adres` don't work too even technically, valid prefixes are used. Do we have a definition somewhere that a valid prefix must be <space><prefix>

Reply:
> Nope tags must be alphanumeric and have no spacing too. So the example u give cannot be used

what about t/NGOtarget/boss

wonder if this will work. But anyways I think onus is on us to state that it won't

Reply:
can't also cus / is not alphanumerical